app/services/get-capacity-view.js

```diff
@@ -74,12 +74,18 @@ var parseCapacityBreakdown = function (workloadReports, organisationLevel) {
 }
 
 var buildCapacityBreakdownEntry = function (workloadReport) {
+  var cmsPercentageValue = null
+
+  if (workloadReport.cmsAdjustmentPoints > 0) {
```

Comment:
🤔  What if we have a net negative adjustments total? Could happen if someone is the recipient of any `-points` values.

Reply:
Myself, Terence and Andy had a chat about this. The decision was that if an OM has had points removed via CMS, then none of their remaining total_points are due to CMS work. So the percentage of their capacity which is being taken up by CMS is 0%. Thoughts?

Reply:
Makes sense 👍